common/display/displayqueue.cpp

```diff
@@ -198,8 +192,7 @@ bool DisplayQueue::QueueUpdate(std::vector<HwcLayer*>& source_layers) {
     overlay_layer.SetIndex(layer_index);
     overlay_layer.SetAcquireFence(layer->acquire_fence.Release());
     queue_item.layers_rects_.emplace_back(layer->GetDisplayFrame());
-    int ret = layer->release_fence.Reset(
-        queue_item.sync_object_->CreateNextTimelineFence());
+    int ret = layer->release_fence.Reset(overlay_layer.GetReleaseFence());
```

Comment:
one more thing here we are resetting the release_fence with the new one. but what is the guarantee that the size of the previous layer and the new set of layers are same. Ideally we should release the previous layers release fence right.